invenio_cli/helpers/docker_compose.py

```diff
@@ -95,3 +103,14 @@ def destroy_containers(self):
 
         # Close logging pipe
         logpipe.close()
+
+    def copy(self, src_file, dst_path, container):
+        """Copy a file into the path of the specified container."""
+        container = self.docker_client.containers.get(container)
+
+        with tarfile.open('tmp.tar', "w") as tar:
+            tar.add(src_file, arcname=os.path.basename(src_file),
+                    recursive=False)
+        with open('tmp.tar', 'rb') as fin:
+            data = io.BytesIO(fin.read())
+            container.put_archive(dst_path, data)
```

Comment:
Looking at https://github.com/docker/docker-py/blob/845d31dbd280f6c28cdb8594c6ff9d9f4b038bea/tests/integration/api_container_test.py#L679 . We should be able to do this (w/o io):

```suggestion
            container.put_archive(dst_path, fin)
```